src/pages/basic_data/institution_admin/edit_user/TasksFormSection.tsx

```diff
@@ -12,29 +10,14 @@ export const rolesWithAreaOfResponsibility = [
   RoleName.NviCurator,
 ];
 
-const StyledChipLabelContainer = styled(Box)({
-  display: 'flex',
-  gap: '0.25rem',
-  alignItems: 'center',
-});
-
 interface TasksFormSectionProps extends AreaOfResponsibilityProps {
   roles?: RoleName[];
-  updateRoles: (roles: RoleName[]) => void;
 }
 
-export const TasksFormSection = ({
-  roles = [],
-  viewingScopes,
-  updateViewingScopes,
-  updateRoles,
-}: TasksFormSectionProps) => {
+export const TasksFormSection = ({ roles = [], viewingScopes, updateViewingScopes }: TasksFormSectionProps) => {
   const { t } = useTranslation();
   const curatorRoleIsAdded = roles.some((rolename) => rolesWithAreaOfResponsibility.includes(rolename));
```

Comment:
Vet ikke, men ville gjetta at thesis curator også skal aktivere valg for ansvarsområde?

![bilde](https://github.com/user-attachments/assets/7881bf65-528a-46f9-9322-8d81702af549)